android_splash_screen/README.md

```diff
@@ -0,0 +1,34 @@
+# Splash Screen Sample
+
+A Flutter app that exemplifies how to implement an animated splash screen for Android devices running at least Android 12, the version that supplies the new [SplashScreen API](https://developer.android.com/about/versions/12/features/splash-screen).
+
+**NOTE:** There is a pub package available to implement static splash screens in your Flutter app: [flutter_native_splash](https://pub.dev/packages/flutter_native_splash).
+
+
+## Goals
+- Demonstrate the compatibility of animated splash screens and Flutter apps running on Android
+- Demonstrate the smoothness achievable by a splash screen as a transition to the Flutter UI
+
+## The Important Bits
+
+### Remove deprecated code
+
+When creating a Flutter app, the Android code generated may include the deprecated implementation of a splash screen. This includes a definition of `io.flutter.embedding.android.SplashScreenDrawable` in the ` /android/app/src/main/AndroidManifest.xml` file and an implementation of `provideSplashScreen()` in the `/android/app/src/main/kotlin/MainActivity.kt` file. Make sure to remove this code.
```

Comment:
Does this mean that `flutter create` is using a template project that includes deprecated Android calls?

Reply:
Yes 😬 but that makes sense because these are only not needed for devices running Android 12.

Reply:
Yes 😬 but this should be an upcoming [change](https://flutter.dev/docs/development/ui/advanced/splash-screen?tab=android-splash-alignment-kotlin-tab#migrating-from-manifest--activity-defined-custom-splash-screens). I can put that in the README